client/src/annoMatrix/whereCache.ts

```diff
@@ -63,16 +63,18 @@ export interface WhereCache {
   };
 }
 
+export type WhereCacheColumnLabels = string[] | number[] | Int32Array;
```

Comment:
This should be (using the new Dataframe types):   ` export type WhereCacheColumnLabels = LabelArray`

It is entirely legal for an array of labels to contain a mix of numbers and strings.

Reply:
Updated [WhereCacheColumnLabels](https://github.com/chanzuckerberg/cellxgene/blob/mim/annoMatrix/client/src/annoMatrix/whereCache.ts#L67).